src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -171,8 +171,10 @@ final case class PurposeApiServiceImpl(
           Future.failed(MissingFreeOfChargeReason)
         else Future.unit
       purpose        <- purposeManagementService.getPurposeById(purposeUUID)
+      _              <- tenantManagementService.getTenantById(purposeUpdateContent.consumerId)
+      _              <- catalogManagementService.getEServiceById(purposeUpdateContent.eserviceId)
```

Comment:
Which is the purpose of these calls?

Reply:
This code checks for the existence of parameters, also mad into the BFF project

Reply:
Ok, let's remove checks on BFF because they are redundant.